.github/workflows/gradle-build-production.yml

```diff
@@ -2,7 +2,7 @@ name: Gradle Build & Deploy - Production
 on:
   push:
     branches:
-      - 'master'
+      - "master"
```

Comment:
any reason why these single quotes were changed ?

Reply:
No that's just prettier formatting from when I added the new env variable in there and saved. Shouldn't change anything but I can revert that if you want